### PR TITLE
Update discord link in https://relay.dev/en/help.html

### PR DESCRIPTION
**Pull request by kafinsalim**
Discord invitation link on the page is invalid.
I'm a member of Reactiflux discord server. #relay channel is deleted, I asked there what channel to ask about relay and another user tells me to ask on #help-graphql channel.
I Also generated the invitation link to that channel that will never expire and for unlimited use.

### CODE REVIEW

**Comment by taion** [comment]
Might make more sense to drop that link entirely? It's unlikely to be too helpful to send Relay users to a generic "GraphQL help" channel from here.

**Comment by kafinsalim** [comment]
@taion I added a commit to drop the discord channel link. Or do you mean to remove all of the discord section? Also I signed the CLA already

File: website/pages/en/help.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Help extends React.Component {
       },
       {
         content:
-          'Many users idle on Discord in [#relay](https://discord.gg/0ZcbPKXt5bX40xsQ) on [Reactiflux](https://www.reactiflux.com/).',
+          'Many users idle on Discord of [Reactiflux](https://www.reactiflux.com/).',
```
Comment (maraisr) [inline website/pages/en/help.js:35]: Think the correct grammer here needs to read: "Many users idle on Discord for Reactiflux.

```suggestion
          'Many users idle on Discord for [Reactiflux](https://www.reactiflux.com/).',
```

**Comment by sibelius** [comment]
what about remove discord and keep only #relay channel of graphql slack?

**Comment by kafinsalim** [comment]
I don't know much about how active relay discussion on Reactiflux's Discord, but a user there says this:

`Id recommend asking relay questions in the #relay channel in graphql's slack
relay users are much more active there`

Then I check #relay channel in graphql slack and it has discussion almost every day so I'm kind of agree 😄
But let's give time to see other people's opinions

**Comment by maraisr** [comment]
I'm leaning on agreeing with @sibelius. Think having a more direct vessel for help is far more effective than having multiple vessels. 

As you've said, the Discrod is a wasteland—and directing people there, might distill some sort of "this project is dead" mentality.

**Comment by kafinsalim** [comment]
Removed discord section and adjust grid layout a bit

Screenshot 2020-10-21 at 11 23 52

**Review by sibelius — APPROVED** [review]

**Comment by poteto** [comment]
Thanks for cleaning this up!

**Pull request closed without merging**